Source/ThirdParty/xdgmime/CMakeLists.txt

```diff
@@ -20,9 +20,26 @@ include_directories("${XDGMIME_INCLUDE_DIRECTORIES}")
 add_definitions(-DXDG_PREFIX=_wk_xdg)
 add_library(xdgmime STATIC ${XDGMIME_SOURCES})
 
+if (CMAKE_VERSION VERSION_LESS "3.20")
+    include(TestBigEndian)
+    test_big_endian(CMAKE_IS_BIG_ENDIAN)
+    if (CMAKE_IS_BIG_ENDIAN)
+        set(CMAKE_CXX_BYTE_ORDER "BIG_ENDIAN")
+    else ()
+        set(CMAKE_CXX_BYTE_ORDER "LITTLE_ENDIAN")
+    endif ()
+endif ()
```

Comment:
Actually, wait: our `CMAKE_MINIMUM_REQUIRED` is 3.20, so none of this is needed! Delete please.

Reply:
You're right, @mcatanzaro, I hadn't noticed. I added this code because I saw the checks for version 3.20 in `OptionsGTK.cmake`.

I grepped the CMake files for version checks and removed the conditional code for CMake < 3.20.

I hope it's okay to do it in the same PR.